target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -799,6 +799,14 @@ tl-wr1043nd-v4)
 	ucidef_set_led_switch "lan3" "LAN3" "tp-link:green:lan3" "switch0" "0x04"
 	ucidef_set_led_switch "lan4" "LAN4" "tp-link:green:lan4" "switch0" "0x02"
 	;;
+tl-wr1043n-v5)
```

Comment:
If we want to merge v4 and v5, we can merge here too, just add the usb for v4 only